configuration/contrib/diag-plugins/07_mosquitto.sh

```diff
@@ -32,19 +33,40 @@ mosquitto_journal() {
 }
 
 mosquitto_log() {
-    if [ -f /var/log/mosquitto/mosquitto.log ]; then 
+    if [ -f /var/log/mosquitto/mosquitto.log ]; then
         cp /var/log/mosquitto/mosquitto.log "$OUTPUT_DIR"/
     else
         echo "mosquitto.log not found" >&2
     fi
 }
 
+mosquitto_config() {
+    if command -V tree >/dev/null >&2; then
+        tree /etc/mosquitto > "$OUTPUT/etc_mosquitto.tree.txt" ||:
```

Comment:
`$OUTPUT` is a mistake I guess.

```suggestion
        tree /etc/mosquitto > "$OUTPUT_DIR/etc_mosquitto.tree.txt" ||:
```